nixos/modules/services/misc/docker-registry.nix

```diff
@@ -96,7 +96,12 @@ in {
 
     extraConfig = lib.mkOption {
       description = ''
-        Docker extra registry configuration via environment variables.
+        Docker extra registry configuration.
+      '';
+      example = lib.literalExpression ''
+        {
+          log.level = "debug";
```

Comment:
You are sure we don't need to quote the left side?

Reply:
```
  services.dockerRegistry.enable = true;
  services.dockerRegistry.extraConfig = {
    log.level = "debug";
  };
```
I took a look at the generated config and it looks good to me:
```json
"log": {
  "fields": {
    "service": "registry"
  },
  "level": "debug"
},
```

Reply:
Just wanted to be sure we don't miss any quoting here